README.md

```diff
@@ -163,12 +163,20 @@ For Linux users `FORGIT_COPY_CMD` should be set to make copy work. Example: `FOR
 Options can be set via environment variables. They have to be **exported** in
 order to be recognized by `forgit`.
 
-For instance, if you want to order branches in `gcb` by the last committed date you could:
+For instance:
+
+- if you want to order branches in `gcb` by the last committed date, you could:
 
 ```shell
 export FORGIT_CHECKOUT_BRANCH_BRANCH_GIT_OPTS='--sort=-committerdate'
 ```
 
+- if you want to see a preview of the stashed items in `gss` preview, you could:
```

Comment:
```suggestion
- if you want to see a overview of the stashed items in `gss` preview, you could:
```

I think __overview__ communicates the options from the example better here, since we already use the term __preview__ for the diff that is being displayed.